app/src/renderer/vuex/modules/transactions.js

```diff
@@ -1,4 +1,3 @@
-import fp from "lodash/fp"
```

Comment:
nice

Reply:
I felt a bit sorry to remove the only example of pure functional stuff XD but that was a too big dependency